lambdatimes/src/components/TopBar.js

```diff
@@ -1,24 +1,75 @@
 import React from 'react';
+import styled from 'styled-components';
 
-// Refactor this component to use styled components and not classNames. 
+// Refactor this component to use styled components and not classNames.
 // You can find the corresponding CSS in the CSS/index.css file
 
```

Comment:
my advice would be to have a separate file for the reusable styles and import them in so this file does not get so cluttered, remember you want to compartmentalize as much as possible